lib/ast-utils/docs.js

```diff
@@ -96,6 +96,51 @@ function annotateElementHeader(descriptor) {
   }
 }
 
+function matchByName(propa, propb) {
+  return propa.name == propb.name;
+}
+
+function copyProperties(from, to, behaviorsByName) {
+  if (from.properties) {
+    from.properties.forEach(function(fromProp){
+      if (fromProp.is in to.properties && !to.properties[fromProp.is].__fromBehavior) {
```

Comment:
If `from.properties` and `to.properties` are both arrays then I think this is unlikely to ever be true, as the `in` operator checks keys, which for an array will be its indexes.